angular-app/src/app/pages/appointments/appointment-details/appointment-details.component.ts

```diff
@@ -38,7 +38,7 @@ export class AppointmentDetailsComponent {
     this.checkIfAuthenticated();
     this.checkIfRegisteredWithVendia();
     this.getAppointments();
-    this.activatedRoute.queryParams.subscribe( data => { this.filterApptName = data['event']; console.log(this.filterApptName); } );
```

Comment:
Thanks for cleaning these console logs up. I always forget to lol

Reply:
there were still tap(console.log), I was not sure how those are used.